src/stylesheets/themes/hugo-terminal-orange/index.scss

```diff
@@ -0,0 +1,24 @@
+@import "./variables";
+@import "../../index";
+@import "./syntax";
+@import "./button.scss";
+@import "./glow.scss";
+@import "./heading.scss";
+@import "./comment.scss";
+@import "./selection.scss";
+@import "./placeholder.scss";
+@import "./combobox.scss";
+@import "./code.scss";
+
+body {
+  background-color: $bg-page;
+}
```

Comment:
is this rule necessary? Thinking setting the $bg-page variable in variables.scss should have the same effect.

Reply:
@jdanyow In my attempt to make it as compliant with other themes  as possible, I copied `github-dark-orange` theme as a base and then edited it. So I'm not really sure :)

Reply:
```scss
body {
  background-color: $bg-page;
}

iframe {
  -webkit-touch-callout: none;
  -webkit-user-select: none;
  -khtml-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
```
**should be deleted from this file**